jupyterhub/values.schema.yaml

```diff
@@ -1559,7 +1559,7 @@ properties:
                 chp:
                   extraCommandLineFlags:
                     - "--auto-rewrite"
-                    - "--custom-header {{ .Values.myCustomStuff }}"
+                    - "--custom-header={{ .Values.custom.myStuff }}"
```

Comment:
Without the `=` replacing the space, this example would lead to "unrecognized flag" where it parsed the full line as the flag name.

And also the Helm chart schema doesn't allow for .Values.myCustomStuff, only for `.Values.custom.myStuff` etc - nested under `custom`.